micromamba/tests/test_create.py

```diff
@@ -143,6 +143,24 @@ def test_env_lockfile_different_install_after_create(tmp_home, tmp_root_prefix,
     helpers.install("-p", env_prefix, "-f", install_spec_file, "-y", "--json")
 
 
+# Only run this test on Linux, as it is the only platform where xeus-cling
+# (which is part of the environment) is available.
+@pytest.timeout(30)
```

Comment:
I think it's:
```suggestion
@pytest.mark.timeout(30)
```
But isn't 30 seconds still a lot? (just checking here ^^)

Reply:
Hum, and is it hanging without the fix even with dry run? (If not the test should be live run).
I would still say that we should have the test actually executed... (my previous question was more asking for confirmation, not really asking for a change).
(And btw, and generally speaking, you can still keep the `assert` even with dry run IINM).

Reply:
> Hum, and is it hanging without the fix even with dry run?

It is not hanging with and without the dry run. It is just that we need nothing else than the dry-run for this test, this would also make it shorter.

I will reintroduce the `assert`.

Reply:
> is if it's hanging with the dry run without the fix

`micromamba` was observed to block during the resolution within `libsolv`. Dry-runs like live-runs (run `libsolv`'s resolution) but just do not perform the transaction then.

I just have rechecked manually and the issue also happens for dry-runs without the fix; why would it not hang in this case?

Reply:
Yes it should, thus the "check" in case we are missing something....
Thanks for checking!